R/graph.decay.curves.R

```diff
@@ -20,14 +20,8 @@
 #' * Setting the  `n_curves` argument to a value smaller than the number of rows in `curve_params` will cause this function to select the first `n_curves` rows to graph.
 #' * Setting `n_curves` larger than the number of rows in ` will result all curves being plotted.
```

Comment:
@d-morrison since `plot_curve_params_one_ab()` is not an exported function but is wrapped by `autoplot.curve_params()`, shoudl some of the text in @details be moved there?